.gitignore

```diff
@@ -12,6 +12,7 @@
 .phpunit.result.cache
 .phpstorm.meta.php
 _ide_helper.php
+composer.lock
```

Comment:
It is not the desired behavior that when running tests from `module-larave`l by commit or PR the CI installs new versions of the modules, even minor ones, because it does not always control possible inconsistencies in the changes in the dependencies, and the CI should not be susceptible to this type of inconvenience. That's why after a local installation tested before commit the `composer.lock` file is sent to the repository. I will revert this change.